lighthouse-core/audits/aria-valid-attr-value.js

```diff
@@ -0,0 +1,64 @@
+/**
+ * @license
+ * Copyright 2016 Google Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+const Audit = require('./audit');
+const Formatter = require('../formatters/formatter');
+
+class ARIAValidAttr extends Audit {
+  /**
+   * @return {!AuditMeta}
+   */
+  static get meta() {
+    return {
+      category: 'Accessibility',
+      name: 'aria-valid-attr-value',
+      description: 'Element aria-* attributes have valid values',
+      requiredArtifacts: ['Accessibility']
+    };
+  }
+
+  /**
+   * @param {!Artifacts} artifacts
+   * @return {!AuditResult}
+   */
+  static audit(artifacts) {
+    const violations = artifacts.Accessibility.violations || [];
+    const rule = violations.find(result => result.id === 'aria-valid-attr-value');
+
+    return ARIAValidAttr.generateAuditResult({
+      rawValue: typeof rule === 'undefined',
+      debugString: this.createDebugString(rule),
+      extendedInfo: {
+        formatter: Formatter.SUPPORTED_FORMATS.ACCESSIBILITY,
+        value: rule
+      }
+    });
+  }
+
+  static createDebugString(rule) {
```

Comment:
This `createDebugString` method begs for a refactor helper :) ...and a lot of these audits.

But for starters, you could factor our `createDebugString` into lib/a11y-helpers.js like I did for common style audit methods:
https://github.com/GoogleChrome/lighthouse/blob/master/lighthouse-core/lib/styles-helpers.js